pyodide_build/buildpkg.py

```diff
@@ -107,6 +107,8 @@ def compile(path, srcpath, pkg, args):
     env = dict(os.environ)
     if pkg.get('build', {}).get('skip_host', True):
         env['SKIP_HOST'] = ''
+    for k, v in pkg.get('build', {}).get('env', {}).items():
+        env[k] = str(v).lower() if type(v) is bool else str(v)
```

Comment:
Could you export setting the env in `meta.yaml` in a separate PR and add it to the documentation?

Reply:
Will do. (I only save the environment to be able to distinguish between "part of configure process" and "necessary for build" (cf. e.g. PYWASMCROSS_SKIP_REPLAY). Will document as such.